Frontend/Pages/Projects/TransferDates/AdvisoryBoard.cshtml.cs

```diff
@@ -65,8 +66,18 @@ public async Task<IActionResult> OnPostAsync()
                 return Page();
             }
 
-            projectResult.Dates.Htb = AdvisoryBoardViewModel.AdvisoryBoardDate.DateInputAsString();
+            projectResult.Dates.Htb = AdvisoryBoardViewModel.AdvisoryBoardDate.DateInputAsString();            
             projectResult.Dates.HasHtbDate = !AdvisoryBoardViewModel.AdvisoryBoardDate.UnknownDate;
+            projectResult.Dates.HtbInserted = DateTime.Now.ToString("dd/MM/yyyy");
+            if(!string.IsNullOrEmpty(projectResult.Dates.HtbInserted))
```

Comment:
To save space you could use the ternary operator
`projectResult.Dates.HasHtbInserted = !string.IsNullOrEmpty(projectResult.Dates.HtbInserted) 
? true
: false`